src/main/java/ratpack/zipkin/ServerTracingModule.java

```diff
@@ -93,26 +101,29 @@ public ClientResponseInterceptor clientResponseInterceptor(final Brave brave) {
   }
 
   @Provides
-  public Brave getBrave(final Config config, final ServerConfig serverConfig) {
-    Brave.Builder braveBuilder = new Brave.Builder(
-        new RatpackServerClientLocalSpanState(
-                buildLocalEndpoint(
-                        config.serviceName,
-                        serverConfig.getPort(),
-                        serverConfig.getAddress()
-                )
-        )
-    );
+  public Brave getBrave(final Tracer tracer) {
+    return TracerAdapter.newBrave(tracer);
```

Comment:
@llinder I don't think this works the way I thought it would - looking at the implementation, it appears to construct a new `Brave` using an instance of `ThreadLocalServerClientAndLocalSpanState`, which obviously isn't much help to us. I had hoped that it would use the `CurrentTraceContext` instance that's already present in the `Tracer`,  but no 🎲